test/behavior/basic.zig

```diff
@@ -16,6 +16,8 @@ test "empty function with comments" {
 }
 
 test "truncate" {
+    if (builtin.zig_backend == .stage2_spirv64) return error.SkipZigTest;
+
```

Comment:
btw, are these tests failing because of vulkan specifically, or because there is something wrong with the backend now? If its the latter, then I suggest fixing them and re-checking the tests. I can send you some patches...

Reply:
it's the latter

Reply:
Update for others following this convo: I updated the spir-v tests for Ali (and fixed a typo in a commit message)